imports/wasi_http/streams/streams.go

```diff
@@ -10,7 +10,10 @@ import (
 	"github.com/tetratelabs/wazero"
 )
 
-const ModuleName = "streams"
+const (
+	ModuleName            = "streams"
+	ModuleName_2023_10_18 = "wasi:io/streams@0.2.0-rc-2023-10-18"
```

Comment:
I have been trying to get this PR working with https://github.com/dev-wasm/dev-wasm-go/tree/main/http without success. The first issue is that module name should be `wasi:io/streams` instead that dev-wasm-go client will work with that. However, still after that there is error

```
panic: wasm: error instantiating guest: "read" is not exported in module "wasi:io/streams"
```

perhaps some mismatch between versions in dev-wasm-go and wasi-go

Reply:
just thinking, could it make sense to make one working combination with wasi-go wasi_http and dev-wasm-go client?

Reply:
@zetaab `dev-wasm-go` unfortunately hasn't been updated (I will do that soon) there is an example here that will work:

https://github.com/brendandburns/hello-wasi-http-go

I'll try to get `dev-wasm-go` updated soon.